library/src/main/java/com/novoda/downloadmanager/lib/DownloadBatch.java

```diff
@@ -0,0 +1,38 @@
+package com.novoda.downloadmanager.lib;
+
+import android.content.ContentValues;
+
+import java.util.List;
+
+public class DownloadBatch {
```

Comment:
package, or do you want clients to be able to create this?  If they get one but can't create on - interface